pkg/terraform/terraform.go

```diff
@@ -271,14 +328,46 @@ func (c *Commander) workspaces() ([]string, error) {
 		return nil, err
 	}
 
-	ws := strings.Split(output.Stdout, "\n")
+	ws := strings.Split(output.Stdout, " ")
 	for i := range ws {
 		ws[i] = strings.TrimSpace(ws[i])
 	}
 
 	return ws, nil
 }
 
+// kubectlCurContext return the cluster name of current context set in kubectl.
+func (c *Commander) kubectlCurContext() (string, error) {
```

Comment:
The method name is too confusing for me. It took a few mins to work out what was intended (p.s. I didn't read the comment and I should've). Perhaps just call it `kubectlContext`?